grayskull/pypi/pypi.py

```diff
@@ -819,26 +837,26 @@ def _extract_requirements(self, metadata: dict) -> dict:
 
         if "pip" not in host_req:
             host_req += [f"python{limit_python}", "pip"]
+        if run_req:
```

Comment:
We shouldn't need this (it should be fixed at `_get_run_req_from_requires_dist` so it always provide a list --even if empty--).

There's another problem here. Instead of testing for truthiness, you check whether is None or not:

```suggestion
        if run_req is not None:
```